share/da.po

```diff
@@ -12,7 +12,7 @@ msgstr ""
 msgid "Print version information and exit."
 msgstr "Udskriv version og afbryd."
 
-msgid "The minimum severity level to display"
+msgid "The minimum severity level to display."
```

Comment:
That is not correct msgid from the code.

Reply:
Yes indeed, I thought the rest of the msgid was wrapped on a second line.